src/shared/awsConsole.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+
+export function getAwsConsoleUrl(service: 'ecr' | 'cloudformation', region: string): vscode.Uri {
+    switch (service) {
+        case 'ecr':
+            return vscode.Uri.parse(`https://${region}.console.aws.amazon.com/ecr/repositories?region=${region}`)
+        case 'cloudformation':
+            return vscode.Uri.parse(`https://${region}.console.aws.amazon.com/cloudformation/home?region=${region}`)
```

Comment:
Not an attempt to "deep link". Just an attempt to provide something useful if there are no CFN stacks or ECR repos.